pkg/crypto/tinkcrypto/wrap_support_test.go

```diff
@@ -134,6 +136,151 @@ func Test_okpKWSupportFailures(t *testing.T) {
 	require.EqualError(t, err, "deriveRecipient1Pu: recipient key not OKP type")
 
 	_, err = okpKW.deriveRecipient1Pu("", nil, nil, []byte{}, []byte{}, []byte{}, 0)
-	require.EqualError(t, err, "deriveRecipient1Pu: derive25519KEK with ephemeral key failed: bad input point:"+
-		" low order point")
+	require.EqualError(t, err, "deriveRecipient1Pu: deriveECDHX25519: bad input point: low order point")
+}
+
+type mockKey struct {
+	Kty string `json:"kty,omitempty"`
+	Crv string `json:"crv,omitempty"`
+	X   string `json:"x,omitempty"`
+	Y   string `json:"y,omitempty"`
+	D   string `json:"d,omitempty"`
+}
+
+type mockRecipient struct {
+	Alg string  `json:"alg,omitempty"`
+	Enc string  `json:"enc,omitempty"`
+	Apu string  `json:"apu,omitempty"`
+	Apv string  `json:"apv,omitempty"`
+	Epk mockKey `json:"epk,omitempty"`
+}
+
+func refJWKtoECKey(t *testing.T, jwkMarshalled []byte) *ecdsa.PrivateKey {
+	t.Helper()
+
+	jwk := &mockKey{}
+	err := json.Unmarshal(jwkMarshalled, jwk)
+	require.NoError(t, err)
+
+	x, err := base64.RawURLEncoding.DecodeString(jwk.X)
+	require.NoError(t, err)
+
+	y, err := base64.RawURLEncoding.DecodeString(jwk.Y)
+	require.NoError(t, err)
+
+	d, err := base64.RawURLEncoding.DecodeString(jwk.D)
+	require.NoError(t, err)
+
+	return &ecdsa.PrivateKey{
+		PublicKey: ecdsa.PublicKey{
+			Curve: elliptic.P256(),
+			X:     new(big.Int).SetBytes(x),
+			Y:     new(big.Int).SetBytes(y),
+		},
+		D: new(big.Int).SetBytes(d),
+	}
+}
+
+var (
```

Comment:
these should be files loaded from a testdata folder.

use Go 1.16 embed to load them.

Reply:
will do..

Reply:
added